Assets/HoloToolkit/Input/Scripts/Microphone/MicStream.cs

```diff
@@ -45,7 +50,8 @@ public enum ErrorCodes { ALREADY_RUNNING = -10, NO_AUDIO_DEVICE, NO_INPUT_DEVICE
 		/// <param name="previewOnDevice">If true, the audio from the microphone will be played through your speakers.</param>
 		/// <param name="micsignal">Optional (can be null): This callback will be called when data is ready for MicGetFrame</param>
 		/// <returns>error code or 0</returns>
-		[DllImport("MicStreamSelector")]    public static extern int MicStartStream(bool keepData, bool previewOnDevice, LiveMicCallback micsignal);
+		[DllImport("MicStreamSelector")]
```

Comment:
Nit: May want to decorate all these imports with ExactSpelling = true to avoid any implicit oopsies.